packages/tables/src/elements/Row.spec.tsx

```diff
@@ -126,7 +114,12 @@ describe('Row', () => {
 
     expect(getByTestId('row')).toHaveStyleRule(
       'background-color',
-      getColorV8('neutralHue', 100, DEFAULT_THEME)
+      getColor({
+        variable: 'background.emphasis',
+        transparency: DEFAULT_THEME.opacity[100],
+        light: { offset: -300 },
+        theme: DEFAULT_THEME
+      })
```

Comment:
Isn't it preferred to reference expected values from PALETTE?

Reply:
Is there an easy way to do that when opacity is in play? I didn't see one. 🤔

Reply:
Oh, nvm, JZ answered it below. :D